tools/check-emfile-handling.js

```diff
@@ -102,6 +102,13 @@ execSync(
 );
 console.log("✅ No errors encountered running ESLint.");
 
+console.log("Running ESLint with --fix...");
+execSync(
+	`node bin/eslint.js ${OUTPUT_DIRECTORY} --fix --no-config-lookup --rule "eol-last: [error, always]"`,
+	{ stdio: "inherit" },
+);
+console.log("✅ No errors encountered running ESLint with --fix.");
```

Comment:
I think it's not necessary to run ESLint twice. We could run it just once with the `--fix` option.

Also, the [`eol-last`](https://eslint.org/docs/latest/rules/eol-last) rule is deprecated, so it's probably better to use a different one for testing. It's also fine to change the content of the generated files to make sure it gets autofixed by one of the rules in the current config:

https://github.com/eslint/eslint/blob/f82e152b4300b067ab5870c73b5f10a8002eaa80/tools/check-emfile-handling.js#L66